frontend/src/modules/resourceprogress/components/ResourceProgress.js

```diff
@@ -92,39 +93,59 @@ export class ResourceProgressBase extends React.Component<Props, State> {
                                 Translated
                             </Localized>
                         </span>
-                        <p className="value">{ asLocaleString(approved) }</p>
+                        <p className="value" onClick={ this.toggleVisibility }>
+                            <Link to={{ search: "?status=translated" }}>
```

Comment:
This still doesn't work, but I'm not sure how best to solve the issue here. I think we'll want to upgrade `react-router` to be able to use a new feature of theirs. Give me some more time, I'll investigate.

Reply:
OK, figured it out. No need to update anything. Just add `pathname` here. To set its value correctly, you'll need to pass `parameters` from `App.js` and reconstitute it in a variable. For example:

```js
const currentPath = `/${params.locale}/${params.project}/${params.resource}/`;
```

Reply:
I will work on this feedback.